### ISSUE TITLE
MenuBar list items are not following high contrast in entire pane.

### ISSUE THREAD

**Issue by StephenLPeters** [microsoft/microsoft-ui-xaml#2594]
Copied from internal [issue](https://microsoft.visualstudio.com/OS/_workitems/edit/26330443?src=WorkItemMention&src-action=artifact_link).

**Pre-requisite: ​**
Turn on High Contrast Mode. (Keyboard Shortcut: Alt+Shift+PrintScreen).

**Repro steps :**
Launch XAML Controls Gallery.
Activate MenuBar from All Controls Page.
Navigate to File/Edit/Help ->Activate it and observe it in high contrast.

**Expected result:**
 List items should change its color according to high contrast.
In HC Black: Background should be Black with White text.
In HC White:Background should be white with black text.

**Actual result:**
List items are appearing in grey background in entire control page.

![HC](https://user-images.githubusercontent.com/7758786/83690397-83582400-a5a5-11ea-864c-d9f72d51a9f8.png)

**Comment by chingucoding** [comment]
I can't seem to repro this using the MUXControlsTestApp:

![gif](https://user-images.githubusercontent.com/16122379/84598027-0ca20d00-ae68-11ea-97c4-f66b636004c7.gif)

Could this be an issue with the XAML Controls Gallery?

**Comment by Chips1234** [comment]
> I can't seem to repro this using the MUXControlsTestApp:
> 
> ![gif](https://user-images.githubusercontent.com/16122379/84598027-0ca20d00-ae68-11ea-97c4-f66b636004c7.gif)
> 
> Could this be an issue with the XAML Controls Gallery?

@chingucoding No, this is in MUXControlsTestApp:
![Menu flyout high contrast bug](https://user-images.githubusercontent.com/47545798/89136850-99715a80-d4ea-11ea-8dea-f6ac3ea5b032.jpg)

**Comment by chingucoding** [comment]
I am not sure what you are referring to @Chips1234 , in the screenshot you posted the background of the flyout seems to be correct.

**Comment by Chips1234** [comment]
> I am not sure what you are referring to @Chips1234 , in the screenshot you posted the background of the flyout seems to be correct.

@chingucoding Ah, I'm sorry. Now I see what the error is.

**Comment by chingucoding** [comment]
@StephenLPeters Can you still repo this issue?

**Comment by StephenLPeters** [comment]
I agree, this doesn't repro for me.
